extension/otto/otto.go

```diff
@@ -261,10 +263,11 @@ func (env *Environment) SetEventTimeLimit(eventRegex string, timeLimit time.Dura
 
 //Clone makes clone of the environment
 func (env *Environment) Clone() ext.Environment {
+	cloneMtx.Lock()
```

Comment:
Are you sure this is required? https://github.com/cloudwan/gohan/blob/2e35ef3cc70937abe33604dd6e8c892b60c9897b/vendor/github.com/robertkrimen/otto/clone.go#L17